packages/labs/analyzer/src/lib/lit-element/properties.ts

```diff
@@ -37,13 +37,14 @@ export const getProperties = (
   let staticProperties;
 
   for (const prop of propertyDeclarations) {
-    if (!ts.isIdentifier(prop.name)) {
+    if (!ts.isIdentifier(prop.name) && !ts.isPrivateIdentifier(prop.name)) {
```

Comment:
@kevinpschaaf questions on this function, or you remember: it's named `getProperties()`, but only returns the reactive properties. 1) Should it be renamed? 2) Since reactive properties are kind of an implementation detail, and a getter/setter pair or different decorator could make a property reactive... are we handling those cases, or should we even distinguish reactive properties from others? Is this to do something else like find attributes?

Reply:
Yeah, iirc this was `properties` in the `@property` sense?  The term `field` is used in the code to refer to generic fields. But yeah could rename to `getReactiveProperties()`.

I think these would be used to determine the `attributes` for e.g. CEM (I see there's still a TODO in there), but also in the Angular wrapper to wrap them with `@Input()`.